clearpath_platform_description/launch/description.launch.py

```diff
@@ -1,68 +1,101 @@
 
-from launch import  LaunchContext, LaunchDescription
-from launch.actions import DeclareLaunchArgument
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, RegisterEventHandler, GroupAction, IncludeLaunchDescription
 from launch.substitutions import Command, FindExecutable, PathJoinSubstitution, LaunchConfiguration
 from launch_ros.actions import Node
 from launch_ros.parameter_descriptions import ParameterValue
 from launch_ros.substitutions import FindPackageShare
+from launch.event_handlers import OnProcessExit
+from launch.launch_description_sources import PythonLaunchDescriptionSource
 
 
 def generate_launch_description():
-    # Launch Configurations
-    robot_model = LaunchConfiguration('robot_model')
-    is_sim = LaunchConfiguration('is_sim')
+    # Packages
+    pkg_clearpath_viz = FindPackageShare('clearpath_viz')
 
+    # Launch Configurations
+    config_file = LaunchConfiguration('config_file', default='/home/rkreinin/clearpath_ws/src/clearpath_config/clearpath_config/sample/a200_config.yaml')
```

Comment:
Same thing about path here.  If it can't be generic, could be something like `/home/administrator/colcon_ws/...`